### PR TITLE
So/erotic stories text

### LINKED ISSUE #216: Replace the placeholder text in Erotic Stories

### PR DESCRIPTION
**Pull request by sssoja**
### What changes have you made?
- replaced the placeholder text with the approved text for both stories 
- applied a `dangerouslySetInnerHTML` on both `ScrollableText`s to render the italics and line breaks 
- included the project icon at the top of the grid in order to be able to nav back to the projects page 

### Which issue(s) does this PR fix?

Fixes #216 

### Screenshots (if there are design changes)

Screenshot 2020-12-16 at 12 13 50

### How to test
Check: 
- the grid displays as one column on mobile
- the grid displays as two columns  and two rows on desktop 
- the text displays with the necessary formatting i.e. italics and line breaks (there's no formatting on the Spanish version of "wings")
- the translation works on both stories 

### Known bugs
The scroll on  `ScrollableText` is not working

### CODE REVIEW

File: src/components/project-pages/EroticStories.tsx [hunk 1]
```diff
@@ -1,20 +1,25 @@
...
+import PreviewOrProjectPage from "./PreviewOrProjectPage";
+import React from "react";
+import styled from "styled-components";
 
 const Main = styled(Flex)<TypographyProps>`
   ${typography};
 `;
 
-const launchDate = "2021-01-21";
+const launchDate = "2020-01-21";
```
Comment (mantagen) [inline src/components/project-pages/EroticStories.tsx:14]: In the past?

**Review by mantagen — COMMENTED** [review]
The layout + formatting look good.

What's the desired outcome for the scrollable text here? (on both mobile and desktop)

File: src/components/project-pages/EroticStories.tsx (hunk 1 above)
Comment (sssoja) [reply src/components/project-pages/EroticStories.tsx]: ah my bad I should have changed it back to 2021, I put the timer back so we could visualise the content - i've updated it now

**Review by Jbarget — APPROVED** [review]

**Comment by sssoja** [comment]
Desired outcome -  good question. I think on mobile this is less clear cut but on desktop what we definitely need is for the text to fit inside its own grid square so that its the same height as the image. The user should be able to scroll the text without having to scroll the page if that makes sense?

**Comment by mantagen** [comment]
@sssoja 

I see. In order for `overflow: scroll` to work, the height needs to be bound in some way. Is it to be bound by the height of the screen? Or a percentage of the screen?

Using `flex: auto;` in this case won't work on it's own, because there's a row below it. If it was just one row, you could say "okay we want this row to fill up all available vertical space and no more" (which i believe we've done elsewhere.

Try giving each row a height of `100vh`, and check it.

Happy to do a code pairing session on it too if that would be helpful

**Comment by sssoja** [comment]
Gotta bind the height, makes sense! That was essentially what we needed. I know it's a little bit hacky but instead of `100vh` I've upped it to `140` because of this: 

Screenshot 2020-12-16 at 15 55 46

when instead we want this: 

Screenshot 2020-12-16 at 15 56 15

I've checked it on mobile and it doesn't appear to have thrown anything off so do you reckon we can go with that?

**Comment by sssoja** [comment]
Okay I'm well and truly stuck can I take you up on that pairing session offer? @mantagen 

I'll push the latest code so you can see where I'm going with this. 

Essentially there was no "row" to add a `vh` to so I've created two of them, one for each of the stories.

**Review by mantagen — APPROVED** [review]

**Comment by mantagen** [comment]
@sssoja you might wanna double check this on safari actualy come to think of it!

**Comment by sssoja** [comment]
@mantagen  10/10 on safari wahooo

**Pull request merged**